back/engines/commercial/multi_tenancy/app/services/multi_tenancy/templates/serializer.rb

```diff
@@ -65,14 +65,43 @@ def run
             'volunteering/volunteer'               => yml_volunteering_volunteers,
             'custom_maps/map_config'               => yml_maps_map_configs,
             'custom_maps/layer'                    => yml_maps_layers,
-            'custom_maps/legend_item'              => yml_maps_legend_items
+            'custom_maps/legend_item'              => yml_maps_legend_items,
+            'content_builder/layout'               => yml_content_builder_layouts,
+            'content_builder/layout_image'         => yml_content_builder_layout_images
```

Comment:
The main concern of this PR: Adding these items (`content_builder` items) to the tenant template serializer.